### ISSUE TITLE
Can't get the solution to run in Ex 9: Transport

### ISSUE THREAD

**Issue by irelandm** [senecajs/seneca-in-practice#39]
My solution and **the provided sample** work fine when invoked from node and tested with curl or a browser, but neither will work when tested with seneca-in-practice.

I get the following error:

> seneca-in-practice run transport.js 

```
/Users/marki/.nvm/versions/node/v0.12.12/lib/node_modules/seneca-in-practice/problems/transport/exercise.js:71
          , exercise.__('fail.connection', {address: url, message: err.message
                     ^
TypeError: undefined is not a function
    at DuplexWrapper.<anonymous> (/Users/marki/.nvm/versions/node/v0.12.12/lib/node_modules/seneca-in-practice/problems/transport/exercise.js:71:22)
    at DuplexWrapper.emit (events.js:129:20)
    at ClientRequest.<anonymous> (/Users/marki/.nvm/versions/node/v0.12.12/lib/node_modules/seneca-in-practice/node_modules/hyperquest/index.js:13:19)
    at ClientRequest.emit (events.js:129:20)
    at Socket.socketErrorListener (_http_client.js:271:9)
    at Socket.emit (events.js:107:17)
    at net.js:459:14
    at process._tickCallback (node.js:355:11)

```

**Comment by adrianmcli** [comment]
Same issue here. I think the exercise is broken. I've been able to run the server and make HTTP requests to it successfully.

**Comment by marcopiraccini** [comment]
That should fix it: https://github.com/senecajs/seneca-in-practice/pull/42

**Comment by marcopiraccini** [comment]
Merged in Released 0.1.14

**Comment by adrianmcli** [comment]
I confirm that this is fixed.
